src/wallet/rpcwallet.cpp

```diff
@@ -4292,6 +4299,13 @@ UniValue z_gettotalbalance(const UniValue& params, bool fHelp)
     if (!EnsureWalletIsAvailable(fHelp))
         return NullUniValue;
 
+    if (!fEnableZGetTotalBalance)
+        throw runtime_error(
+            "z_gettotalbalance is DEPRECATED and will be removed in a future release\n\n"
+            "Use z_getbalanceforaccount instead, or restart with `-allowdeprecated=z_gettotalbalance`\n"
```

Comment:
This API returned transparent balance as well:
```suggestion
            "Use z_getbalanceforaccount or getbalance instead, or restart with `-allowdeprecated=z_gettotalbalance`\n"
```

Also, I'm not sure `z_getbalanceforaccount` is the correct alternative, because for existing wallets they will have legacy shielded addresses which can't be queried via that API. We probably need to recommend `z_getbalanceforviewingkey` here instead or as well.

(Update the help text to match whatever this ends up saying.)